returnn/search.py

```diff
@@ -160,6 +160,7 @@ def create_returnn_config(self, returnn_config, log_verbosity, device):
         res = copy.deepcopy(returnn_config)
         res.config = config
         res.post_config = post_config
+        res.check_consistency()
```

Comment:
Wouldn't it be better to call `check_consistency()` only internally in ReturnnConfig when it's constructed and when config or post_config are modified?
It's not a client's responsibility to check that everything is correct

Reply:
This is a bit too much, if we construct dozends of configs with hundreds of keys that will slow down config loading. So if it's done this would be a good place to do it. But I dont think it should be done at all.